app/src/main/java/org/bspb/smartbirds/pro/backend/dto/Location.java

```diff
@@ -80,51 +80,8 @@ public ContentValues toCV() {
         cv.put(_ID, id);
         cv.put(LAT, latitude);
         cv.put(LON, longitude);
-        cv.put(NAME_BG, name != null ? name.bg : null);
-        cv.put(NAME_EN, name != null ? name.en : null);
```

Comment:
why loose the columns?

Reply:
Because we have data column with the whole json and these columns are not used anymore.

Reply:
@onyxvd but the data column is not new, so it seems there was a reason for these columns to exists. In any case - if you checked they are not actually used, i'm fine with removing. Because it's a bit out of scope for the PR and wanted to make sure it's not incidental.